tests/test_forcing_nudging.py

```diff
@@ -0,0 +1,238 @@
+"""
+Tests for checking different options related to nudging when doing forcing
+conversion to
+
+In this file we will be doing two things:
+
+    1) Checking that the forcing step runs without error for different valid
+    sets of parameters in the yaml-files that describe the conversion step
+    2) When converting to DEPHY format we also check the output xr.Dataset
```

Comment:
I don't think we do this currently. Simply remove?

Reply:
yes, let's remove this